CHEATSHEET.md

```diff
@@ -151,25 +160,21 @@ If what you are interested in is not directly returned because for example the f
 
 Hooking
 
-```python
-hook(addr, hook, length=0, kwargs=None)
-```
-
-There are already predefined hooks for libc.so.6 functions (useful for staticly compiled libraries)
+There are already predefined hooks for libc functions (useful for statically compiled libraries)
 
 ```python
-hook = simuvex.SimProcedures['libc.so.6']['atoi']
-hook(addr, hook, length=4, kwargs=None)
+proj = angr.Project('/path/to/binary', use_sim_procedures=True)
+proj.hook(addr, angr.SIM_PROCEDURES['libc']['atoi']())
 ```
 
 Hooking with Simprocedure:
 
 ```python
-class fixpid(SimProcedure):
+class fixpid(angr.SimProcedure):
     def run(self):
             return 0x30
 
-b.hook(0x4008cd, fixpid, length=5)
+proj.hook(0x4008cd, fixpid, length=5)
```

Comment:
should be `fixpid()`

Reply:
...now that I think of it, why is there `length=5` here? That doesn't do anything in this context...